api.json

```diff
@@ -3082,6 +2969,7 @@
     "/apps/{app_id}/users/{external_user_id}": {
       "put": {
         "operationId": "update_player_tags",
+        "deprecated": true,
```

Comment:
for this and all of these, can we describe what to use instead of the now deprecated endpoints?  I don't think there's a dedicated field we could use to describe this, but maybe in the `description`?